### PR TITLE
Project restructuring

### PR DESCRIPTION
**Pull request by Thodor12**
## Changes proposed:
- The current structure has a few problems which this PR aims to resolve:
   - Root level files (like crowdin.yml) are unnecessarily published because the Jekyll source root is `.`.
   - Testing locally is not possible because the Jekyll binaries are not provided, I've added a `GemFile` into the project and a short description in the README on how to setup Ruby/Bundler. This should allow people to test locally when making changes so they can verify if what they are providing actually works.
- Despite these changes the Jekyll _config.yml remains in the root so deployment CI should not be affected.

(I recommend not checking all changed files, review the like 5 top most files which actually change something, the rest are all files moved)

Merge please :D

### CODE REVIEW

**Comment by Thodor12** [comment]
2nd attempt, however this time we first verify the deployment CI

**Comment by Thodor12** [comment]
@OrionDevelopment Could you review the changes to the Dockerfile and docker-compose.
I was able to run a local image using:
```
docker build -t test .
docker run -it --rm -v 'D:\Projects\MinecoloniesWiki:/srv/jekyll' -p "4000:4000" test
```
I just reflected this by adding the volume in the compose, I've not touched the ports since it seems like traefik already handles this.

**Review by OrionDevelopment — CHANGES_REQUESTED** [review]
You can not use volumes in a cluster setup

File: docker-compose.yml [hunk 1]
```diff
@@ -3,8 +3,8 @@ version: '3.8'
 services:
   jekyll-host:
     image: container.ldtteam.com/ldtteam/minecolonies/wiki:latest
-    volumes:
-      - .:/srv/jekyll
+    ports:
```
Comment (OrionDevelopment) [inline docker-compose.yml:6]: Don't expose ports in the compose.
This is a runtime security risk and not needed.

If you need to modify the port on which our cluster ingress access the container, you need to modify the labels attached to the container.

**Review by OrionDevelopment — CHANGES_REQUESTED** [review]

**Review by OrionDevelopment — APPROVED** [review]
Infrastructure wise this should be fine now.

**Pull request merged**